src/components/protection/ClaimBnt.vue

```diff
@@ -14,16 +14,13 @@
         :src="bntLogoSrc"
         alt="Token Logo"
       />
-      <span class="mx-2">{{ `${prettifyNumber(item.amount)} BNT` }}</span>
-      <!-- <span class="text-primary font-size-12">
-        {{ `(~$${item.usdValue})` }}
-      </span> -->
+      <span class="amt-num mx-2">{{ `${prettifyNumber(item.amount)} BNT` }}</span>
     </div>
     <div v-if="!locked">
       <b-btn
         variant="primary"
         @click="click"
-        class="font-size-14 font-w500 px-4"
+        class="btn-claim font-size-14 font-w500 px-4"
```

Comment:
`btn-claim` class doesnt seem to exist

Reply:
just set for testing, I'll try to find it without specific class name.